build.gradle

```diff
@@ -22,11 +22,11 @@ allprojects {
 }
 
 ext {
-    compileSdkVersion = 25
+    compileSdkVersion = 26
     buildToolsVersion = "25.0.3"
```

Comment:
As before, shouldn't we build with something starting with 26? Here and in travis file... No?